week-1/L-functions-nested/exercise.js

```diff
@@ -3,3 +3,13 @@ var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+
+function shout() {
+  var x = ["Daniel", "Irina", "Mimi", "Rob", "Yohannes"];
+  var size = x.length;
+  for (i = 0; i < size; i++) {
+    var upperName = x[i].toUpperCase();
+    console.log("HELLO " + upperName);
+  }
+}
+var y = shout();
```

Comment:
Why assign a function to a variable?

Reply:
I think I could call it without assigning it. like that shout();